tools/svg-to-font.js

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env node
+const svgDir='build/svg-clean';
+const codepointsFile = 'src/font/codepoints.json';
+const destFontDir='build/font';
+const cssFile = destFontDir + '/social-logos.css';
+const woff2FontFile = destFontDir + '/social-logos.woff2';
+
+const process = require('process');
+const fs = require('fs');
+const path = require('path');
+const glob = require('glob');
+const SVGIcons2SVGFontStream = require('svgicons2svgfont');
+const svg2ttf = require('svg2ttf');
+const wawoff2 = require('wawoff2');
+
+// Start in the right folder.
+const rootDir = __dirname + '/..';
+process.chdir(rootDir);
+
+const getCodepoint = name => {
+	// If a codepoint for this name does not yet exist, create one.
+	if (!codepoints[name]) {
+		maxCodepoint++;
+		codepoints[name] = maxCodepoint;
+	}
+	// Return codepoint.
+	return codepoints[name];
+}
+
+const writeCodepoints = () => {
+	fs.writeFile(codepointsFile, JSON.stringify(codepoints, null, 4), {encoding: 'utf8'}, err => {
+		if(err) throw err;
+		// console.log('Wrote codepoints file.');
+	});
+}
+
+const svg2woff2 = async (fontBuffer) => {
+	const ttf = svg2ttf(fontBuffer.toString(), {});
```

Comment:
A note for future possible improvement: Every build produces a slightly different font file because the ttf format embeds a timestamp (and that carries through to the woff2). If we could determine a better timestamp to pass as `ts` here (maybe `git log -1 --first-parent --format=%ct src/svg/*.svg`? 😀), that could make the builds more repeatable.

There are a few other fields we could supply here too, like `description` and `url` and `version`.

Reply:
> Every build produces a slightly different font file because the ttf format embeds a timestamp

The changed file was on my list to investigate. Thanks for digging and coming up with a solution before me! 😅

Reply:
Implemented in bc37718. I ended up using `git log -1 --first-parent --format=%ct src/svg` (no wildcard) so it captures commits that remove files as well.